workshops/20_python_beginners_workshop/tasks/short_tasks/practice_boolean.md

```diff
@@ -0,0 +1,27 @@
+# Exercise 2: Boolean & Comparisons
+
+This is the way how you usually write comments in Python:
+
+~~~Python
+# Hi! I'm Anna and I live in practice_boolean.md.
+# I'm an expert Python coder.
+# I'm 21 years old and I plan to program cool stuff forever.
+~~~
+
+Comments are useful for additional explanation of your code.
+
+### Instructions:
+
+**1.**
+Someone has introduced themselves to you using comments in practice_boolean.py.
```

Comment:
I think this can be a link to the file `practice_boolean.py`

Reply:
Check https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links

Reply:
I do not understand if this file is supposed to be created by them or not

Reply:
I do not use a link. The file is not supposed to be created by them.